### DISCUSSION TITLE
Not work

### DISCUSSION THREAD

**Discussion by QusaiImayyah** [QuestPDF/QuestPDF#424] [General]
Canvas((canvas, size) =>
{
    using var paint = new SKPaint
    {
        Color = SKColors.Red,
        StrokeWidth = 10,
        IsStroke = true
    };

    // move origin to the center of the available space
    canvas.Translate(size.Width / 2, size.Height / 2);

    // draw a circle
    canvas.DrawCircle(0, 0, 50, paint);
});

**Comment by DawidRobertNowak** [comment]
Hello,

We are closing this thread due to inactivity. Unfortunately, there was not enough detail provided for us to assist. Should you have a specific question or need help with a particular use case, please feel free to open a new discussion with additional context.

Thank you!
